src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1256,31 +1262,37 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
         }
 
         // Returns true if the status code is one of the supported redirection codes.
-        private static bool IsRedirectCode(HttpStatusCode code)
+        private static bool IsRedirectCode(HttpStatusCode statusCode)
         {
-            int intCode = (int)code;
-            return
-            (
-                (intCode >= 300 && intCode < 304) ||
-                intCode == 307 ||
-                intCode == 308
-            );
+            switch (statusCode)
```

Comment:
If you like you could use switch expression here and below.

Reply:
done

Reply:
I don't see the change - it is not switch expression.

Reply:
I'm sorry, I had done it on an offline branch